pipeline/gke-deploy-build-push/0.1/README.md

```diff
@@ -3,10 +3,21 @@
 
 This Pipeline builds, pushes, and deploys your application to a Google Kubernetes Engine cluster using [`gke-deploy`](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gke-deploy).
 
+## Install the task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/pipeline/gke-deploy-build-push/0.1/support/kaniko-build.yaml
+```
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/pipeline/gke-deploy-build-push/0.1/support/gke-deploy.yaml
```

Comment:
probably we can just pass the install URL for `gke-deploy` task over here instead of taking the task from support dir
```
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/gke-deploy/0.1/gke-deploy.yaml
```